wire-ios/WireCommonComponents/Icons/Autogenerated/WireStyleKit.swift

```diff
@@ -4384,7 +4385,7 @@ public class WireStyleKit: NSObject {
 
         //// Bezier Drawing
         context.saveGState()
-        context.setAlpha(0.24)
+        context.setAlpha(0.40)
```

Comment:
Why do we need these changes?

Reply:
So we can fix the color/opacity for the color of the icon properly.